docs/coordinates/matchsep.rst

```diff
@@ -73,9 +72,9 @@ rather than the single scalar angular offset of a separation.
 offsets encountered in astronomy.
 
 The first piece of such functionality is the
-:meth:`~astropy.coordinates.SkyCoord.position_angle` method, which gives the
-conventional astronomy position angle (positive angles East of North) from one
-the |skycoord| it is called on to another given as the argument::
+:meth:`~astropy.coordinates.SkyCoord.position_angle` method. This method
+computes the position angle (positive angles East of North) between one
```

Comment:
(this suggestion is meant to be paired with the other one on the line just below this one)
```suggestion
computes the position angle between one
```